src/cli/cli_commissioner.cpp

```diff
@@ -59,6 +74,27 @@ template <> otError Commissioner::Process<Cmd("announce")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli commissioner energy
+ * @code
+ * commissioner energy 0x00050000 2 32 1000 fdde:ad00:beef:0:0:ff:fe00:c00
+ * Done
+ * Energy: 00050000 0 0 0 0
+ * @endcode
+ * @cparam commissioner energy @ca{mask} @ca{count} @ca{period} @ca{scanDuration} @ca{destination}
+ *   * `mask`: Bitmask that identifies channels for performing IEEE 802.15.4 ED Scans.
+ *   * `count`: Number of IEEE 802.15.4 ED Scans per channel.
+ *   * `period`: Number of milliseconds between successive IEEE 802.15.4 ED Scans.
+ *   * `scanDuration`: IEEE 802.15.4 ScanDuration in milliseconds to use when
+ *     performing an IEEE 802.15.4 ED Scan.
+ *   * `destination`: Destination IPv6 address for the message. The message may be multicast.
+ * @par
+ * Sends an Energy Scan Query message.
+ * @par
+ * The contents of `MGMT_ED_REPORT` messages,
```

Comment:
MGMT_ED_REPORT messages are not really explained here, I have no clue what they are.  Should this be reworded to note that things are printed as received without getting into too many details?